tests/extension/oneapi_enqueue_functions/enqueue_functions.cpp

```diff
@@ -0,0 +1,397 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Copyright (c) 2024 The Khronos Group Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+*******************************************************************************/
+
+#include "../../common/common.h"
+
+namespace enqueue_functions::tests {
+
+#ifdef SYCL_EXT_ONEAPI_ENQUEUE_FUNCTIONS
+namespace syclex = sycl::ext::oneapi::experimental;
```

Comment:
Small nit, but I would prefer to rename this to something like `oneapi_ext` or similar, to make it clear it's a oneapi ext.